TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robotSubSystems/elevator/Elevator.java

```diff
@@ -48,6 +48,8 @@ public static void operate(ElevatorStates elevatorState, Gamepad gamepad1) {
             case OVERRIDE:
                 elevatorPower = gamepad1.right_stick_y;
                 break;
+            case DEPLETE:
+                elevatorPower = ElevatorConstants.depletePower;
```

Comment:
I don't think deplete should be it's own elevator state, I think it should be part of override, and then when we are depleting we are going to override and adding -0.1 to what the driver is asking.

Reply:
there will be a problem since you will be have to add another condition and in it to call elevator.operate once again
do you prefer this option?

Reply:
Robot state should be global, then you can check if we are depleting.